### PR TITLE
API conventions cleanup for 2.2

### PR DESCRIPTION
**Pull request by scottaddie**
Addresses https://github.com/aspnet/Docs/issues/9134

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/web-api/advanced/conventions?view=aspnetcore-2.2&branch=pr-en-us-9761)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@scottaddie 	FYI `WIP` label no longer triggers WIP validation. It needs to be in the title.

**Comment by scottaddie** [comment]
@Rick-Anderson Should we delete the WIP label? Since it no longer works with the WIP app, I can't think of a purpose that it serves.

**Comment by Rick-Anderson** [comment]
@scottaddie 	yes. cc @tdykstra 	@guardrex

**Comment by scottaddie** [comment]
@Rick-Anderson That WIP label has been deleted from the repo.

File: aspnetcore/web-api/advanced/conventions.md [hunk 1]
```diff
@@ -3,40 +3,60 @@ title: Use web API conventions
...
+By [Pranav Krishnamoorthy](https://github.com/pranavkm) and [Scott Addie](https://github.com/scottaddie)
 
-By default, ASP.NET Core MVC 2.2 ships with a set of default conventions, `Microsoft.AspNetCore.Mvc.DefaultApiConventions`. The conventions are based on the controller that ASP.NET Core scaffolds. If your actions follow the pattern that scaffolding produces, you should be successful using the default conventions.
+ASP.NET Core 2.2 introduces a way to extract common [API documentation](xref:tutorials/web-api-help-pages-using-swagger) and apply it to multiple actions, controllers, or all controllers within an assembly. Web API conventions are a substitute for decorating individual actions with [[ProducesResponseType]](xref:Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute).
```
Comment (Rick-Anderson) [inline aspnetcore/web-api/advanced/conventions.md]: Let's get this ready for 3.0 release
ASP.NET Core 2.2 and later includes a way to ...

File: aspnetcore/web-api/advanced/conventions.md [hunk 2]
```diff
@@ -3,40 +3,60 @@ title: Use web API conventions
...
 
-At runtime, <xref:Microsoft.AspNetCore.Mvc.ApiExplorer> understands conventions. `ApiExplorer` is MVC's abstraction to communicate with Open API document generators. Attributes from the applied convention get associated with an action and are included in the action's Swagger documentation. API analyzers also understand conventions. If your action is unconventional (for example, it returns a status code that isn't documented by the applied convention), it produces a warning, encouraging you to document it.
+A convention allows you to:
+
+* Define the most common return types and status codes returned from a specific type of action.
+* Identify actions that deviate from the defined standard.
+
+ASP.NET Core MVC 2.2 includes a set of default conventions in `Microsoft.AspNetCore.Mvc.DefaultApiConventions`. The conventions are based on the controller that ASP.NET Core scaffolds. If your actions follow the pattern that scaffolding produces, you should be successful using the default conventions.
```
Comment (Rick-Anderson) [inline aspnetcore/web-api/advanced/conventions.md]: ASP.NET Core MVC 2.2 **and later** includes a set of default conventions

Comment (Rick-Anderson) [reply aspnetcore/web-api/advanced/conventions.md]: > The conventions are based on the controller that ASP.NET Core scaffolds.

I don't understand that. That seems to contradict ASP.NET Core MVC 2.2 includes a set of default conventions in `Microsoft.AspNetCore.Mvc.DefaultApiConventions`.

Can you explain how scaffolding determines conventions? Why do you need to **follow the pattern that scaffolding produces**? Isn't that just a good example?

File: aspnetcore/web-api/advanced/conventions.md [hunk 3]
```diff
@@ -3,40 +3,60 @@ title: Use web API conventions
...
+Conventions don't compose; each action may be associated with exactly one convention. More specific conventions take precedence over less specific conventions. The selection is non-deterministic when two or more conventions of the same priority apply to an action. The following options exist to apply a convention to an action, from the most specific to the least specific:
+
+1. `Microsoft.AspNetCore.Mvc.ApiConventionMethodAttribute` &mdash; Applies to individual actions and specifies the convention type and the convention method that applies.
 
-1. `Microsoft.AspNetCore.Mvc.ApiConventionMethodAttribute` &mdash; Applies to individual actions and specifies the convention type and the convention method that applies. In the following sample, the convention method `Microsoft.AspNetCore.Mvc.DefaultApiConventions.Put` is applied to the `Update` action:
+    In the following example, the convention method `Microsoft.AspNetCore.Mvc.DefaultApiConventions.Put` is applied to the `Update` action:
```
Comment (Rick-Anderson) [inline aspnetcore/web-api/advanced/conventions.md]: What's the Put convention? What does this do? Maybe something like:

 In the following example, the convention method `Microsoft.AspNetCore.Mvc.DefaultApiConventions.Put` is applied to the `Update` action. This convention ...

**Pull request merged**